src/main/java/org/broadinstitute/hellbender/utils/bwa/BwaMemIndex.java

```diff
@@ -185,6 +230,7 @@ private static void loadNativeLibrary() {
         }
     }
 
+    private static native boolean createReferenceIndex(String referenceName, String indexPrefix, String algorithmName);
```

Comment:
Rather than creating a new method, I'd prefer to see createIndexImageFile extended to include this new functionality.